src/main/kotlin/com/hibob/academy/employeeFeedback/dao/FeedbackDao.kt

```diff
@@ -44,6 +45,16 @@ class FeedbackDao(private val sql: DSLContext) {
             .where(feedbackTable.companyId.eq(companyId))
             .fetch(feedbackMapper)
 
+    fun viewFeedbackOfEmployee(employeeId: Int, companyId: Int): List<FeedbackInfo> =
+        sql.select(feedbackTable.id, feedbackTable.employeeId,
```

Comment:
note that viewFeedbackOfEmployee and getFeedbackById are very similar.
Also think about the additional filtering you will need to support.

Try to remove the code repeat

Reply:
new pr- talked about it offline